tests/vmi_multus_test.go

```diff
@@ -876,29 +887,197 @@ var _ = Describe("[Serial]SRIOV", func() {
 	})
 })
 
+var _ = Describe("[Serial]Macvtap", func() {
+	var err error
+	var virtClient kubecli.KubevirtClient
+	var macvtapLowerDevice string
+	var macvtapNetworkName string
+
+	tests.BeforeAll(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		tests.PanicOnError(err)
+
+		macvtapLowerDevice = "eth0"
+		macvtapNetworkName = "net1"
+
+		// cleanup the environment
+		tests.BeforeTestCleanup()
+	})
+
+	BeforeEach(func() {
+		tests.EnableFeatureGate(virtconfig.MacvtapGate)
```

Comment:
If we get rid of the `tets.BeforeAll` we still cannot remove the `[Serial]`since we update the global kubevirt configuration. Updating the global config is not allowed for parallel tests. (`EnableFeatureGate` -> `UpdateKubeVirtConfigValueAndWait` -> [fail](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L4193) if in the parallel e2e suite).
 
You can do what the other feature gated features do - [enable it by default for all the tests](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L804) and [skip](https://github.com/kubevirt/kubevirt/blob/115c51ff8a5bb365a6f0874f9c91b984ca85411d/tests/migration_test.go#L147) the tests if the feature is not enabled.

Roman also suggested to add a serial test which explicitly disables the feature gate and tests if the behaviour without it is right. I couldn't find any example for it for the other feature we enable by default. @rmohr do we have any?

Reply:
I'll address enabling this whole test file to run in parallel in a follow-up PR.

Reply:
That's treu for the `AfterEach` but why do we need separate `BeforeEach`es?

Reply:
I prefer to have a single resource being setup in each `BeforeEach` block; I think it becomes more readable.

Let me know if you want me to change it.